dask_ml/decomposition/incremental_pca.py

```diff
@@ -135,12 +136,18 @@ def __init__(
     ):
         self.n_components = n_components
         self.whiten = whiten
+        self.center = center
         self.copy = copy
         self.batch_size = batch_size
         self.svd_solver = svd_solver
         self.iterated_power = iterated_power
         self.random_state = random_state
 
+    def _check_params(self):
+        super()._check_params()
```

Comment:
What is the purpose of defining an empty `_check_params()` on the parent class and then calling it here? In my opinion, that is adding a bit of additional indirection for no functional benefit. Could you please remove this line and the parent class's empty `_check_params()`?

Reply:
The purpose is to future-proof this by ensuring any checks that get eventually introduced in the superclass, get considered at this level as well.